packages/core/contracts-ovm/insured-bridge/implementation/L1_BridgeRouter.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.7.6;
+
+import "@eth-optimism/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol";
+
+import "./L2_BridgeDepositBox.sol";
+
+/**
+ * @notice Contract deployed on L1 that has an implicit reference to a DepositBox on L2 and provides methods for
+ * "Relayers" to fulfill deposit orders to that contract. The Relayers can either post capital to fulfill the deposit
+ * instantly, or request that the funds are taken out of the passive liquidity provider pool following a challenge period.
+ * @dev A "Deposit" is an order to send capital from L2 to L1, and a "Relay" is a fulfillment attempt of that order.
```

Comment:
Note this point carefully: I designate all orders sent from L2 as "Deposits". Any attempt to fulfill is a deposit is a "Relay".

Reply:
great terminology 👌